### PR TITLE
Improve "generating test data" SOP

### PR DESCRIPTION
**Pull request by idazucchi**
I'm suggesting these changes to the SOP based on the process of generating test data for the treatment protocol (ebi-ait/hca-ebi-wrangler-central#635) and cell-to-cell suspension (ebi-ait/hca-ebi-wrangler-central#927).

Let me know if you would like to add something else!

### CODE REVIEW

**Review by Wkt8 — APPROVED** [review]
LGTM!

**Review by amnonkhen — DISMISSED** [review]
please link related ticket.
Link format is: org/repo#num where num is the issue or pr number.

**Review by ESapenaVentura — APPROVED** [review]
LGTM

**Review by MightyAx — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,5 +29,6 @@ When asked for a project UUID, enter the project uuid of the exported project. R
 8. A new folder within `src/` will be created, called `tests`. Delete the root `tests` folder and move the `src/tests` folder to the root of the repository
 9. Delete the newly created `src/temp_data` folder
 10. Push the changes made to this branch to Github
-11. Create a PR from the branch to master - this will include the modified test data, and the diff should show only the files that have been changed as a result of the metadata schema change 
-12. Ensure you update the excel spreadsheet located in the schema-test-data repo with the up-to-date modified spreadsheet for the next cycle of generating test data 
+11. Ensure you update the excel spreadsheet located in the schema-test-data repo with the up-to-date modified spreadsheet for the next cycle of generating test data
```
Comment (Wkt8) [inline README.md]:
```suggestion
11. Ensure you update the excel spreadsheet (Schema_Test_Data.xlsx) located in the schema-test-data repo with the up-to-date modified spreadsheet for the next cycle of generating test data 
```

**Review by Wkt8 — CHANGES_REQUESTED** [review]
Change to specify the excel document! LGTM otherwise.

**Review by Wkt8 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,7 +3,11 @@ This repository is for the test [meta]data associated with the latest updates of
 
 # Brief SOP on generating test data 
 
-1. Download the excel spreadsheet located in this folder (schema_test_data.xlsx) and modify it to include the new metadata-schema changes. **PLEASE REMEMBER TO INCLUDE THE NEW DATA (IF ANY) IN THE HCA-UTIL AREA STATED IN THE NEXT STEP**
+1. Download the excel spreadsheet located in this folder (schema_test_data.xlsx) and modify it to include the new metadata-schema changes.
```
Comment (amnonkhen) [inline README.md]:
```suggestion
1. Download the excel spreadsheet located in this folder ([Schema_Test_Data.xlsx](./Schema_Test_Data.xlsx)) and modify it to include the new metadata-schema changes.
```

**Pull request merged**